### ISSUE TITLE
Layer 7 Protocol column shows up blank under Current Connections

### ISSUE THREAD

**Issue by user501254** [ericpaulbishop/gargoyle#607]
Device Name:TL-WR740Nv4
Gargoyle Version:1.9.2
Model:TP-Link TL-WR740N/ND v4
Device Configuration:Gateway

**Comment by lantis1008** [comment]
Layer 7 does not function in the current version

Avoid setting any rules for it or your router will reboot continuously

**Comment by user501254** [comment]
Hi @lantis1008 

It didn't work on stable version 1.8.1 either. is this planned for a future release?

**Comment by lantis1008** [comment]
Yes it has been broken for some time. I know it worked in 1.6.2. 

I intend to fix it for a future release, but it is a non trivial change.
